client/src/index.js

```diff
@@ -1,8 +1,247 @@
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Link from '@material-ui/core/Link';
+import Snackbar from '@material-ui/core/Snackbar';
+
+const styles = makeStyles(theme => ({
+    root: {
+        height: '100vh',
+    },
+    paper: {
+        margin: theme.spacing(25, 4),
+        isplay: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+    },
+    image: {
+        backgroundImage: 'url(https://images.unsplash.com/photo-1493770348161-369560ae357d?ixlib=rb-1.2.1&auto=format&fit=crop&w=1000&q=80)',
+        backgroundRepeat: 'no-repeat',
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+    },
+    button: {
+        margin: theme.spacing(3, 0, 2),
+    },
+    form: {
+        width: '100%', // Fix IE 11 issue.
+        marginTop: theme.spacing(1),
+    },
+  }));
+
+export default function Login() {
+
+    //error slide up
+    const [open, setOpen] = React.useState(false);
+    const [transition, setTransition] = React.useState(undefined);
+
+
+    const state = {
+        email: "",
+        password: "",
+        confirmPassword: "",
+        firstname: "",
+        lastname: "",
+        isLogin: true
+    };
+
+    const updateEmail = e => {
+        state.email = e.target.value
+    }
+    
+    const updatePassword = e => {
+        state.password = e.target.value
+    }
+
+    const updateConfirmPassword = e => {
+        state.confirmPassword = e.target.value
+    }
+
+    const updateFirstname = e => {
+        state.firstname = e.target.value
+    }
+
+    const updateLastname = e => {
+        state.lastname = e.target.value
+    }
+    
+    const handleClose = () => {
+        setOpen(false);
+    };
+
+    const handleSubmit = () => {
+        //distinguish between login and signup
+        console.log(state.isLogin)
+        //Login
+        if (state.isLogin) {
+            if (document.getElementById("standard-username").value === "" || document.getElementById("standard-password").value === "") {
+                //setTransition();
+                //setOpen(true);
+                console.log("Please fill in all requirements!!!")
+            } else {
+                //login successfully
+            }
+        } else {
+            //Sign up
+            if (document.getElementById("standard-username").value === "" || 
+                document.getElementById("standard-password").value === "" ||
+                document.getElementById("firstname").value === "" ||
+                document.getElementById("lastname").value === "" ||
+                document.getElementById("confirmPassword").value === "") {
+                //setTransition();
+                //setOpen(true);
+                console.log("Please fill in all requirements!!!")
+            } else {
+                //sign up successfully
+            }
+        }
+    }
+
+    const switchToSignupPage = () => {
```

Comment:
Can we add something here to reset the state when switching to the signup page? If the user enters a username or password, what they entered will transfer over if the user clicks `Create a new account`

Reply:
Fixed